src/css/mapbox-gl.css

```diff
@@ -769,3 +769,27 @@ a.mapboxgl-ctrl-logo.mapboxgl-compact {
         display: none;
     }
 }
+
+.mapboxgl-scroll-zoom-blocker-control {
+    color: #fff;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
+    font-size: 100%;
+    justify-content: center;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    z-index: 3;
+    pointer-events: none;
+    transition: opacity 0.1s ease-in-out;
```

Comment:
Just a nitpick, in case someone has a large font size or a narrow map.

<img width="100" alt="Screen Shot 2021-09-20 at 11 38 15 AM" src="https://user-images.githubusercontent.com/572717/134056466-1ef40c30-ec26-4268-a037-9d00e1d4c2a4.png">

```suggestion
    transition: opacity 0.1s ease-in-out;
    text-align: center;
```